app/components/forms/events/view/exhibitors/exhibitor-form.ts

```diff
@@ -52,7 +52,7 @@ export default class ExhibitorForm extends Component<Args> {
           optional : true,
           rules    : [
             {
-              type   : 'regExp',
+              type   : 'email',
```

Comment:
not working

Reply:
https://user-images.githubusercontent.com/42090582/150650324-38fa4607-f2b2-4a21-967a-aa04e3ba2b67.mp4

Reply:
@pc-beast